### PR TITLE
Replace topup modal with topup route

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-react/src/App.tsx [hunk 1]
```diff
@@ -97,24 +94,45 @@ export const App = () => {
...
               path="/topup"
+              render={() =>
+                <div>
+                  <Link to="/topUpRegular">Regular</Link><br />
+                  <Link to="/topUpDeployment">Deployment</Link>
+                </div>}
+            />
+            <Route
+              exact
+              path="/topupRegular"
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/App.tsx]: Should be `topUp...`

File: universal-login-wallet/src/ui/react/Home/HomeScreen.tsx [hunk 2]
```diff
@@ -54,6 +49,15 @@ const HomeScreen = () => {
                   className="jarvis-backup"
                 />
               </Route>
+              <Route exact path="/topup">
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/src/ui/react/Home/HomeScreen.tsx]: `topUp`

File: universal-login-wallet/src/ui/react/Home/HomeScreen.tsx [hunk 3]
```diff
@@ -34,7 +29,7 @@ const HomeScreen = () => {
               <Route path="/" exact>
                 <Funds
                   deployedWallet={walletService.getDeployedWallet()}
-                  onTopUpClick={() => modalService.showModal('topUpAccount', topUpProps)}
+                  onTopUpClick={() => history.push('/topup')}
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/src/ui/react/Home/HomeScreen.tsx]:
```suggestion
                  onTopUpClick={() => history.push('/topUp')}
```

**Comment by cehali** [comment]
Something is wrong with styles, in the first picture title should be aligned to left and the background colour is wrong. In the second and the third there isn't gasPrice component and in the third logos of payment method are not loaded.

image
image
image

**Pull request merged**